.pre-commit-config.yaml

```
@@ -2,14 +2,14 @@
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
   - repo: https://github.com/ambv/black
```

Part of the psf nowadays :)

```suggestion
  - repo: https://github.com/psf/black
```

---

Oh thanks =)